packages/redux-routine/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ### Bug Fixes
 
-- Fix unhandled promise rejection error caused by returning null from registered generator ([#13314](https://github.com/WordPress/gutenberg/pull/13314)
+- Fix unhandled promise rejection error caused by returning null from registered generator ([#13314](https://github.com/WordPress/gutenberg/pull/13314))
+- Removed `castError`.  The middleware will now simply throw the value exposed on a Promise reject or thrown error rather than coercing to an instance of `Error`.  Consuming code can now access whatever value is thrown directly.
```

Comment:
`castError` was never exposed to a consumer, so they needn't care about it, at least by name.

```suggestion
- The middleware will no longer attempt to coerce an error to an instance of `Error`, and instead passes through the thrown value directly.
```